CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 ### Added
 
 ### Changed
+- Bumped the version of Node used by Travis to v6
+- Removing custom grid variables.
```

Comment:
Minor: Should go under removal, or maybe be updated to explain what the resulting effect of that action is, or both.